packages/valory/skills/liquidity_rebalancing_abci/rounds.py

```diff
@@ -188,16 +187,14 @@ def end_block(self) -> Optional[Tuple[BaseSynchronizedData, Event]]:
 
             strategy = json.loads(self.most_voted_payload)
 
-            event = Event.RESET_TIMEOUT
             if strategy["action"] == StrategyType.WAIT.value:
-                event = Event.DONE
-            elif strategy["action"] == StrategyType.ENTER.value:
-                event = Event.DONE_ENTER
-            elif strategy["action"] == StrategyType.EXIT.value:
-                event = Event.DONE_EXIT
-            elif strategy["action"] == StrategyType.SWAP_BACK.value:
-                event = Event.DONE_SWAP_BACK
-            return synchronized_data, event
+                return synchronized_data, Event.DONE
+            if strategy["action"] == StrategyType.ENTER.value:
+                return synchronized_data, Event.DONE_ENTER
+            if strategy["action"] == StrategyType.EXIT.value:
+                return synchronized_data, Event.DONE_EXIT
+            if strategy["action"] == StrategyType.SWAP_BACK.value:
+                return synchronized_data, Event.DONE_SWAP_BACK
```

Comment:
Not sure why we explicitly returned a timeout event here.

Reply:
not sure why we use `.value` here either, inconsistent with the rest

Reply:
I guess so that we do not pass the enum object through the payload. Instead, we pass the value and compare it with the other values.

A mapping from value to event would simplify the code here, but this is out of the scope of this PR.